src/context.rs

```diff
@@ -82,18 +82,7 @@ impl Context {
 
     /// Create a new context, including the standard library
     pub fn new() -> Context {
-        let mut ctx = Context::empty();
-
-        // Include the standard library
-        let mut stdlib_incl =
-            crate::instruction::Incl::new(String::from("stdlib"), Some(String::from("")));
-        stdlib_incl.set_base(PathBuf::new());
-        // FIXME: Remove unwrap
-        ctx.entry_point
-            .add_instruction(Box::new(stdlib_incl))
-            .unwrap();
-
-        ctx
+        Context::empty()
```

Comment:
I think we should remove entirely one of the functions, either `new` or `empty`. This can be done in this PR or in another one

Reply:
I agree. I would keep `new` (seems more idiomatic I think). I will update this PR as soon as possible.